src/server/service/kv.rs

```diff
@@ -845,14 +844,14 @@ fn response_batch_commands_request<F>(
         }
         Ok(())
     });
-    executor.spawn(f);
+    let _ = executor.spawn(f);
```

Comment:
Should we handle error if spawn failed?

Reply:
Spawn failure means the TiKV is about to shutdown.

Reply:
Got it